INFO:openflow.flow_tracker:Set QueryInterval:1.0 LinkMaxBw:19.0Mbps LinkCongThreshold:13.0Mbps AvgSmoothFactor:0.5 LogPeakUsage:1.0
INFO:openflow.flow_tracker:Writing flow tracker info to file: flowtracker_15-15-23_November-23_2015.txt
INFO:misc.groupflow_event_tracer:Writing event trace info to file: eventtrace_15-15-23_November-23_2015.txt
INFO:misc.benchmark_terminator:Module initialized.
INFO:openflow.groupflow:Set link weight type: 1
INFO:openflow.groupflow:Set StaticLinkWeight:1.0 UtilLinkWeight:10.0
INFO:openflow.groupflow:Set FlowReplacementMode:0 FlowReplacementInterval:15.0 seconds
WARNING:openflow.of_01:<class 'pox.openflow.PacketIn'> raised on dummy OpenFlow nexus
INFO:openflow.flow_tracker:Network peak link throughput (Mbps): 0 Time:1448320526.09
INFO:openflow.flow_tracker:Network peak link throughput (Mbps): 0 Time:1448320526.76
INFO:openflow.flow_tracker:Network peak link throughput (Mbps): 0 Time:1448320527.42
WARNING:openflow.igmp_manager:1 not found in ports of router: 00-00-00-00-00-04
INFO:openflow.flow_tracker:Network peak link throughput (Mbps): 0 Time:1448320528.12
INFO:openflow.flow_tracker:Network avg link throughput (Mbps): 0.0 Time:1448320528.12
INFO:openflow.flow_tracker:Network peak link throughput (Mbps): 0.00216967805025 Time:1448320528.81
INFO:openflow.flow_tracker:Network avg link throughput (Mbps): 0.000542419512563 Time:1448320528.81
INFO:openflow.flow_tracker:Network peak link throughput (Mbps): 0.00108483902513 Time:1448320529.48
INFO:openflow.flow_tracker:Network avg link throughput (Mbps): 0.000271209756282 Time:1448320529.48
INFO:openflow.flow_tracker:Network peak link throughput (Mbps): 0.00108483902513 Time:1448320530.14
INFO:openflow.flow_tracker:Network avg link throughput (Mbps): 0.000331198972506 Time:1448320530.14
INFO:openflow.flow_tracker:Network peak link throughput (Mbps): 0.000806802664181 Time:1448320530.82
INFO:openflow.flow_tracker:Network avg link throughput (Mbps): 0.000430508893823 Time:1448320530.82
INFO:openflow.flow_tracker:Network peak link throughput (Mbps): 0.000806802664181 Time:1448320531.51
INFO:openflow.flow_tracker:Network avg link throughput (Mbps): 0.000430508893823 Time:1448320531.51
INFO:openflow.flow_tracker:Network peak link throughput (Mbps): 0.000555614408122 Time:1448320532.17
INFO:openflow.flow_tracker:Network avg link throughput (Mbps): 0.000291413868041 Time:1448320532.17
INFO:openflow.flow_tracker:Network peak link throughput (Mbps): 0.000336388648923 Time:1448320532.84
INFO:openflow.flow_tracker:Network avg link throughput (Mbps): 0.000244866762045 Time:1448320532.84
INFO:openflow.flow_tracker:Network peak link throughput (Mbps): 0.000336388648923 Time:1448320533.51
INFO:openflow.flow_tracker:Network avg link throughput (Mbps): 0.000217033984345 Time:1448320533.51
INFO:openflow.flow_tracker:Network peak link throughput (Mbps): 0.000168194324461 Time:1448320534.18
INFO:openflow.flow_tracker:Network avg link throughput (Mbps): 0.000122433381023 Time:1448320534.18
INFO:openflow.flow_tracker:Network peak link throughput (Mbps): 0.000459180577827 Time:1448320534.85
INFO:openflow.flow_tracker:Network avg link throughput (Mbps): 0.000367207418241 Time:1448320534.85
INFO:openflow.flow_tracker:Network peak link throughput (Mbps): 0.000459180577827 Time:1448320535.56
INFO:openflow.flow_tracker:Network avg link throughput (Mbps): 0.000367207418241 Time:1448320535.56
WARNING:openflow.flow_tracker:PortStats unavailable for Switch: 00-00-00-00-00-08 Port: None
WARNING:openflow.flow_tracker:PortStats unavailable for Switch: 00-00-00-00-00-08 Port: None
WARNING:openflow.flow_tracker:PortStats unavailable for Switch: 00-00-00-00-00-08 Port: None
WARNING:openflow.flow_tracker:PortStats unavailable for Switch: 00-00-00-00-00-08 Port: None
WARNING:openflow.flow_tracker:PortStats unavailable for Switch: fe-e9-eb-5e-90-45 Port: None
WARNING:openflow.flow_tracker:PortStats unavailable for Switch: 00-00-00-00-00-05 Port: None
WARNING:openflow.flow_tracker:PortStats unavailable for Switch: 00-00-00-00-00-05 Port: None
WARNING:openflow.groupflow:Path could not be determined for receiver 00-00-00-00-00-01 (network is not fully connected)
WARNING:openflow.groupflow:Path could not be determined for receiver 00-00-00-00-00-06 (network is not fully connected)
WARNING:openflow.groupflow:Path could not be determined for receiver 00-00-00-00-00-07 (network is not fully connected)
WARNING:openflow.groupflow:Path could not be determined for receiver 00-00-00-00-00-07 (network is not fully connected)
INFO:openflow.flow_tracker:Termination signalled, closed log file: flowtracker_15-15-23_November-23_2015.txt
WARNING:openflow.igmp_manager:Link down: 00-00-00-00-00-02 -> 00-00-00-00-00-01
WARNING:openflow.groupflow:Multicast topology changed, recalculating all paths.
WARNING:openflow.flow_tracker:PortStats unavailable for Switch: 00-00-00-00-00-02 Port: None
WARNING:openflow.flow_tracker:PortStats unavailable for Switch: fe-e9-eb-5e-90-45 Port: None
WARNING:openflow.flow_tracker:PortStats unavailable for Switch: 00-00-00-00-00-08 Port: None
WARNING:openflow.flow_tracker:PortStats unavailable for Switch: 00-00-00-00-00-08 Port: None
WARNING:openflow.flow_tracker:PortStats unavailable for Switch: 00-00-00-00-00-08 Port: None
WARNING:openflow.flow_tracker:PortStats unavailable for Switch: 00-00-00-00-00-08 Port: None
WARNING:openflow.flow_tracker:PortStats unavailable for Switch: 00-00-00-00-00-05 Port: None
WARNING:openflow.flow_tracker:PortStats unavailable for Switch: 00-00-00-00-00-05 Port: None
WARNING:openflow.groupflow:Path could not be determined for receiver 00-00-00-00-00-01 (network is not fully connected)
WARNING:openflow.groupflow:Path could not be determined for receiver 00-00-00-00-00-06 (network is not fully connected)
WARNING:openflow.groupflow:Path could not be determined for receiver 00-00-00-00-00-07 (network is not fully connected)
WARNING:openflow.groupflow:Path could not be determined for receiver 00-00-00-00-00-07 (network is not fully connected)
WARNING:openflow.groupflow:Could not get connection for router: 00-00-00-00-00-01
WARNING:openflow.igmp_manager:Router down: 00-00-00-00-00-01
WARNING:openflow.groupflow:Multicast topology changed, recalculating all paths.
WARNING:openflow.flow_tracker:PortStats unavailable for Switch: 00-00-00-00-00-02 Port: None
WARNING:openflow.flow_tracker:PortStats unavailable for Switch: fe-e9-eb-5e-90-45 Port: None
WARNING:openflow.flow_tracker:PortStats unavailable for Switch: 00-00-00-00-00-08 Port: None
WARNING:openflow.flow_tracker:PortStats unavailable for Switch: 00-00-00-00-00-08 Port: None
WARNING:openflow.flow_tracker:PortStats unavailable for Switch: 00-00-00-00-00-08 Port: None
WARNING:openflow.flow_tracker:PortStats unavailable for Switch: 00-00-00-00-00-08 Port: None
WARNING:openflow.flow_tracker:PortStats unavailable for Switch: 00-00-00-00-00-05 Port: None
WARNING:openflow.flow_tracker:PortStats unavailable for Switch: 00-00-00-00-00-05 Port: None
WARNING:openflow.groupflow:Path could not be determined for receiver 00-00-00-00-00-01 (network is not fully connected)
WARNING:openflow.groupflow:Path could not be determined for receiver 00-00-00-00-00-06 (network is not fully connected)
WARNING:openflow.groupflow:Path could not be determined for receiver 00-00-00-00-00-07 (network is not fully connected)
WARNING:openflow.groupflow:Path could not be determined for receiver 00-00-00-00-00-07 (network is not fully connected)
WARNING:openflow.groupflow:Could not get connection for router: 00-00-00-00-00-01
WARNING:openflow.igmp_manager:Link down: 00-00-00-00-00-04 -> 00-00-00-00-00-02
WARNING:openflow.groupflow:Multicast topology changed, recalculating all paths.
WARNING:openflow.flow_tracker:PortStats unavailable for Switch: 00-00-00-00-00-02 Port: None
WARNING:openflow.flow_tracker:PortStats unavailable for Switch: fe-e9-eb-5e-90-45 Port: None
WARNING:openflow.flow_tracker:PortStats unavailable for Switch: 00-00-00-00-00-08 Port: None
WARNING:openflow.flow_tracker:PortStats unavailable for Switch: 00-00-00-00-00-08 Port: None
WARNING:openflow.flow_tracker:PortStats unavailable for Switch: 00-00-00-00-00-08 Port: None
WARNING:openflow.flow_tracker:PortStats unavailable for Switch: 00-00-00-00-00-08 Port: None
WARNING:openflow.flow_tracker:PortStats unavailable for Switch: 00-00-00-00-00-05 Port: None
WARNING:openflow.flow_tracker:PortStats unavailable for Switch: 00-00-00-00-00-05 Port: None
WARNING:openflow.groupflow:Path could not be determined for receiver 00-00-00-00-00-01 (network is not fully connected)
WARNING:openflow.groupflow:Path could not be determined for receiver 00-00-00-00-00-06 (network is not fully connected)
WARNING:openflow.groupflow:Path could not be determined for receiver 00-00-00-00-00-07 (network is not fully connected)
WARNING:openflow.groupflow:Path could not be determined for receiver 00-00-00-00-00-07 (network is not fully connected)
WARNING:openflow.groupflow:Could not get connection for router: 00-00-00-00-00-01
WARNING:openflow.igmp_manager:Link down: fe-e9-eb-5e-90-45 -> 00-00-00-00-00-02
WARNING:openflow.groupflow:Multicast topology changed, recalculating all paths.
WARNING:openflow.flow_tracker:PortStats unavailable for Switch: 00-00-00-00-00-02 Port: None
ERROR:core:Exception while handling Discovery!LinkEvent...
Traceback (most recent call last):
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 231, in raiseEventNoErrors
    return self.raiseEvent(event, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 278, in raiseEvent
    rv = event._invoke(handler, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 156, in _invoke
    return handler(self, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/openflow/igmp_manager.py", line 1033, in _handle_LinkEvent
    self.raiseEvent(MulticastTopoEvent(MulticastTopoEvent.LINK_DOWN, link_changes, self.adjacency))
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 278, in raiseEvent
    rv = event._invoke(handler, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 156, in _invoke
    return handler(self, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 608, in _handle_MulticastTopoEvent
    self.multicast_paths[multicast_addr][source].update_flow_placement(groupflow_trace_event)
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 375, in update_flow_placement
    self.calc_path_tree_dijkstras(groupflow_trace_event)
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 128, in calc_path_tree_dijkstras
    self._calc_link_weights()
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 200, in _calc_link_weights
    sys.float_info.max / core.openflow_flow_tracker.get_num_tracked_links())
ZeroDivisionError: float division by zero
WARNING:openflow.igmp_manager:Link down: 00-00-00-00-00-02 -> 00-00-00-00-00-04
WARNING:openflow.groupflow:Multicast topology changed, recalculating all paths.
WARNING:openflow.flow_tracker:PortStats unavailable for Switch: 00-00-00-00-00-02 Port: None
ERROR:core:Exception while handling Discovery!LinkEvent...
Traceback (most recent call last):
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 231, in raiseEventNoErrors
    return self.raiseEvent(event, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 278, in raiseEvent
    rv = event._invoke(handler, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 156, in _invoke
    return handler(self, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/openflow/igmp_manager.py", line 1033, in _handle_LinkEvent
    self.raiseEvent(MulticastTopoEvent(MulticastTopoEvent.LINK_DOWN, link_changes, self.adjacency))
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 278, in raiseEvent
    rv = event._invoke(handler, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 156, in _invoke
    return handler(self, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 608, in _handle_MulticastTopoEvent
    self.multicast_paths[multicast_addr][source].update_flow_placement(groupflow_trace_event)
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 375, in update_flow_placement
    self.calc_path_tree_dijkstras(groupflow_trace_event)
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 128, in calc_path_tree_dijkstras
    self._calc_link_weights()
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 200, in _calc_link_weights
    sys.float_info.max / core.openflow_flow_tracker.get_num_tracked_links())
ZeroDivisionError: float division by zero
WARNING:openflow.igmp_manager:Link down: 00-00-00-00-00-02 -> 00-00-00-00-00-0a
WARNING:openflow.groupflow:Multicast topology changed, recalculating all paths.
WARNING:openflow.flow_tracker:PortStats unavailable for Switch: fe-e9-eb-5e-90-45 Port: None
ERROR:core:Exception while handling Discovery!LinkEvent...
Traceback (most recent call last):
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 231, in raiseEventNoErrors
    return self.raiseEvent(event, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 278, in raiseEvent
    rv = event._invoke(handler, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 156, in _invoke
    return handler(self, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/openflow/igmp_manager.py", line 1033, in _handle_LinkEvent
    self.raiseEvent(MulticastTopoEvent(MulticastTopoEvent.LINK_DOWN, link_changes, self.adjacency))
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 278, in raiseEvent
    rv = event._invoke(handler, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 156, in _invoke
    return handler(self, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 608, in _handle_MulticastTopoEvent
    self.multicast_paths[multicast_addr][source].update_flow_placement(groupflow_trace_event)
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 375, in update_flow_placement
    self.calc_path_tree_dijkstras(groupflow_trace_event)
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 128, in calc_path_tree_dijkstras
    self._calc_link_weights()
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 200, in _calc_link_weights
    sys.float_info.max / core.openflow_flow_tracker.get_num_tracked_links())
ZeroDivisionError: float division by zero
WARNING:openflow.igmp_manager:Link down: 00-00-00-00-00-02 -> fe-e9-eb-5e-90-45
WARNING:openflow.groupflow:Multicast topology changed, recalculating all paths.
WARNING:openflow.flow_tracker:PortStats unavailable for Switch: fe-e9-eb-5e-90-45 Port: None
ERROR:core:Exception while handling Discovery!LinkEvent...
Traceback (most recent call last):
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 231, in raiseEventNoErrors
    return self.raiseEvent(event, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 278, in raiseEvent
    rv = event._invoke(handler, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 156, in _invoke
    return handler(self, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/openflow/igmp_manager.py", line 1033, in _handle_LinkEvent
    self.raiseEvent(MulticastTopoEvent(MulticastTopoEvent.LINK_DOWN, link_changes, self.adjacency))
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 278, in raiseEvent
    rv = event._invoke(handler, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 156, in _invoke
    return handler(self, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 608, in _handle_MulticastTopoEvent
    self.multicast_paths[multicast_addr][source].update_flow_placement(groupflow_trace_event)
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 375, in update_flow_placement
    self.calc_path_tree_dijkstras(groupflow_trace_event)
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 128, in calc_path_tree_dijkstras
    self._calc_link_weights()
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 200, in _calc_link_weights
    sys.float_info.max / core.openflow_flow_tracker.get_num_tracked_links())
ZeroDivisionError: float division by zero
WARNING:openflow.igmp_manager:Router down: 00-00-00-00-00-02
WARNING:openflow.groupflow:Multicast topology changed, recalculating all paths.
WARNING:openflow.flow_tracker:PortStats unavailable for Switch: fe-e9-eb-5e-90-45 Port: None
ERROR:core:Exception while handling OpenFlowNexus!ConnectionDown...
Traceback (most recent call last):
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 231, in raiseEventNoErrors
    return self.raiseEvent(event, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 278, in raiseEvent
    rv = event._invoke(handler, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 156, in _invoke
    return handler(self, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/openflow/igmp_manager.py", line 1119, in _handle_ConnectionDown
    self.raiseEvent(event)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 278, in raiseEvent
    rv = event._invoke(handler, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 156, in _invoke
    return handler(self, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 608, in _handle_MulticastTopoEvent
    self.multicast_paths[multicast_addr][source].update_flow_placement(groupflow_trace_event)
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 375, in update_flow_placement
    self.calc_path_tree_dijkstras(groupflow_trace_event)
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 128, in calc_path_tree_dijkstras
    self._calc_link_weights()
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 200, in _calc_link_weights
    sys.float_info.max / core.openflow_flow_tracker.get_num_tracked_links())
ZeroDivisionError: float division by zero
WARNING:openflow.igmp_manager:Link down: fe-e9-eb-5e-90-45 -> 00-00-00-00-00-03
WARNING:openflow.groupflow:Multicast topology changed, recalculating all paths.
WARNING:openflow.flow_tracker:PortStats unavailable for Switch: 00-00-00-00-00-08 Port: None
ERROR:core:Exception while handling Discovery!LinkEvent...
Traceback (most recent call last):
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 231, in raiseEventNoErrors
    return self.raiseEvent(event, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 278, in raiseEvent
    rv = event._invoke(handler, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 156, in _invoke
    return handler(self, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/openflow/igmp_manager.py", line 1033, in _handle_LinkEvent
    self.raiseEvent(MulticastTopoEvent(MulticastTopoEvent.LINK_DOWN, link_changes, self.adjacency))
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 278, in raiseEvent
    rv = event._invoke(handler, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 156, in _invoke
    return handler(self, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 608, in _handle_MulticastTopoEvent
    self.multicast_paths[multicast_addr][source].update_flow_placement(groupflow_trace_event)
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 375, in update_flow_placement
    self.calc_path_tree_dijkstras(groupflow_trace_event)
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 128, in calc_path_tree_dijkstras
    self._calc_link_weights()
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 200, in _calc_link_weights
    sys.float_info.max / core.openflow_flow_tracker.get_num_tracked_links())
ZeroDivisionError: float division by zero
WARNING:openflow.igmp_manager:Link down: 00-00-00-00-00-05 -> 00-00-00-00-00-03
WARNING:openflow.groupflow:Multicast topology changed, recalculating all paths.
WARNING:openflow.flow_tracker:PortStats unavailable for Switch: 00-00-00-00-00-08 Port: None
ERROR:core:Exception while handling Discovery!LinkEvent...
Traceback (most recent call last):
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 231, in raiseEventNoErrors
    return self.raiseEvent(event, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 278, in raiseEvent
    rv = event._invoke(handler, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 156, in _invoke
    return handler(self, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/openflow/igmp_manager.py", line 1033, in _handle_LinkEvent
    self.raiseEvent(MulticastTopoEvent(MulticastTopoEvent.LINK_DOWN, link_changes, self.adjacency))
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 278, in raiseEvent
    rv = event._invoke(handler, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 156, in _invoke
    return handler(self, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 608, in _handle_MulticastTopoEvent
    self.multicast_paths[multicast_addr][source].update_flow_placement(groupflow_trace_event)
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 375, in update_flow_placement
    self.calc_path_tree_dijkstras(groupflow_trace_event)
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 128, in calc_path_tree_dijkstras
    self._calc_link_weights()
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 200, in _calc_link_weights
    sys.float_info.max / core.openflow_flow_tracker.get_num_tracked_links())
ZeroDivisionError: float division by zero
WARNING:openflow.igmp_manager:Router down: 00-00-00-00-00-03
WARNING:openflow.groupflow:Multicast topology changed, recalculating all paths.
WARNING:openflow.flow_tracker:PortStats unavailable for Switch: 00-00-00-00-00-08 Port: None
ERROR:core:Exception while handling OpenFlowNexus!ConnectionDown...
Traceback (most recent call last):
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 231, in raiseEventNoErrors
    return self.raiseEvent(event, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 278, in raiseEvent
    rv = event._invoke(handler, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 156, in _invoke
    return handler(self, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/openflow/igmp_manager.py", line 1119, in _handle_ConnectionDown
    self.raiseEvent(event)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 278, in raiseEvent
    rv = event._invoke(handler, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 156, in _invoke
    return handler(self, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 608, in _handle_MulticastTopoEvent
    self.multicast_paths[multicast_addr][source].update_flow_placement(groupflow_trace_event)
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 375, in update_flow_placement
    self.calc_path_tree_dijkstras(groupflow_trace_event)
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 128, in calc_path_tree_dijkstras
    self._calc_link_weights()
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 200, in _calc_link_weights
    sys.float_info.max / core.openflow_flow_tracker.get_num_tracked_links())
ZeroDivisionError: float division by zero
WARNING:openflow.igmp_manager:Link down: 00-00-00-00-00-08 -> 00-00-00-00-00-04
WARNING:openflow.groupflow:Multicast topology changed, recalculating all paths.
WARNING:openflow.flow_tracker:PortStats unavailable for Switch: 00-00-00-00-00-08 Port: None
ERROR:core:Exception while handling Discovery!LinkEvent...
Traceback (most recent call last):
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 231, in raiseEventNoErrors
    return self.raiseEvent(event, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 278, in raiseEvent
    rv = event._invoke(handler, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 156, in _invoke
    return handler(self, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/openflow/igmp_manager.py", line 1033, in _handle_LinkEvent
    self.raiseEvent(MulticastTopoEvent(MulticastTopoEvent.LINK_DOWN, link_changes, self.adjacency))
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 278, in raiseEvent
    rv = event._invoke(handler, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 156, in _invoke
    return handler(self, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 608, in _handle_MulticastTopoEvent
    self.multicast_paths[multicast_addr][source].update_flow_placement(groupflow_trace_event)
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 375, in update_flow_placement
    self.calc_path_tree_dijkstras(groupflow_trace_event)
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 128, in calc_path_tree_dijkstras
    self._calc_link_weights()
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 200, in _calc_link_weights
    sys.float_info.max / core.openflow_flow_tracker.get_num_tracked_links())
ZeroDivisionError: float division by zero
WARNING:openflow.igmp_manager:Router down: 00-00-00-00-00-04
WARNING:openflow.groupflow:Multicast topology changed, recalculating all paths.
WARNING:openflow.flow_tracker:PortStats unavailable for Switch: 00-00-00-00-00-08 Port: None
ERROR:core:Exception while handling OpenFlowNexus!ConnectionDown...
Traceback (most recent call last):
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 231, in raiseEventNoErrors
    return self.raiseEvent(event, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 278, in raiseEvent
    rv = event._invoke(handler, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 156, in _invoke
    return handler(self, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/openflow/igmp_manager.py", line 1119, in _handle_ConnectionDown
    self.raiseEvent(event)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 278, in raiseEvent
    rv = event._invoke(handler, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 156, in _invoke
    return handler(self, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 608, in _handle_MulticastTopoEvent
    self.multicast_paths[multicast_addr][source].update_flow_placement(groupflow_trace_event)
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 375, in update_flow_placement
    self.calc_path_tree_dijkstras(groupflow_trace_event)
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 128, in calc_path_tree_dijkstras
    self._calc_link_weights()
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 200, in _calc_link_weights
    sys.float_info.max / core.openflow_flow_tracker.get_num_tracked_links())
ZeroDivisionError: float division by zero
WARNING:openflow.igmp_manager:Link down: 00-00-00-00-00-05 -> 00-00-00-00-00-09
WARNING:openflow.groupflow:Multicast topology changed, recalculating all paths.
WARNING:openflow.flow_tracker:PortStats unavailable for Switch: 00-00-00-00-00-08 Port: None
ERROR:core:Exception while handling Discovery!LinkEvent...
Traceback (most recent call last):
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 231, in raiseEventNoErrors
    return self.raiseEvent(event, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 278, in raiseEvent
    rv = event._invoke(handler, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 156, in _invoke
    return handler(self, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/openflow/igmp_manager.py", line 1033, in _handle_LinkEvent
    self.raiseEvent(MulticastTopoEvent(MulticastTopoEvent.LINK_DOWN, link_changes, self.adjacency))
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 278, in raiseEvent
    rv = event._invoke(handler, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 156, in _invoke
    return handler(self, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 608, in _handle_MulticastTopoEvent
    self.multicast_paths[multicast_addr][source].update_flow_placement(groupflow_trace_event)
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 375, in update_flow_placement
    self.calc_path_tree_dijkstras(groupflow_trace_event)
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 128, in calc_path_tree_dijkstras
    self._calc_link_weights()
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 200, in _calc_link_weights
    sys.float_info.max / core.openflow_flow_tracker.get_num_tracked_links())
ZeroDivisionError: float division by zero
WARNING:openflow.igmp_manager:Router down: 00-00-00-00-00-05
WARNING:openflow.groupflow:Multicast topology changed, recalculating all paths.
WARNING:openflow.flow_tracker:PortStats unavailable for Switch: 00-00-00-00-00-08 Port: None
ERROR:core:Exception while handling OpenFlowNexus!ConnectionDown...
Traceback (most recent call last):
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 231, in raiseEventNoErrors
    return self.raiseEvent(event, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 278, in raiseEvent
    rv = event._invoke(handler, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 156, in _invoke
    return handler(self, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/openflow/igmp_manager.py", line 1119, in _handle_ConnectionDown
    self.raiseEvent(event)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 278, in raiseEvent
    rv = event._invoke(handler, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 156, in _invoke
    return handler(self, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 608, in _handle_MulticastTopoEvent
    self.multicast_paths[multicast_addr][source].update_flow_placement(groupflow_trace_event)
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 375, in update_flow_placement
    self.calc_path_tree_dijkstras(groupflow_trace_event)
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 128, in calc_path_tree_dijkstras
    self._calc_link_weights()
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 200, in _calc_link_weights
    sys.float_info.max / core.openflow_flow_tracker.get_num_tracked_links())
ZeroDivisionError: float division by zero
WARNING:openflow.igmp_manager:Link down: 00-00-00-00-00-08 -> 00-00-00-00-00-06
WARNING:openflow.groupflow:Multicast topology changed, recalculating all paths.
WARNING:openflow.flow_tracker:PortStats unavailable for Switch: 00-00-00-00-00-08 Port: None
ERROR:core:Exception while handling Discovery!LinkEvent...
Traceback (most recent call last):
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 231, in raiseEventNoErrors
    return self.raiseEvent(event, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 278, in raiseEvent
    rv = event._invoke(handler, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 156, in _invoke
    return handler(self, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/openflow/igmp_manager.py", line 1033, in _handle_LinkEvent
    self.raiseEvent(MulticastTopoEvent(MulticastTopoEvent.LINK_DOWN, link_changes, self.adjacency))
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 278, in raiseEvent
    rv = event._invoke(handler, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 156, in _invoke
    return handler(self, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 608, in _handle_MulticastTopoEvent
    self.multicast_paths[multicast_addr][source].update_flow_placement(groupflow_trace_event)
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 375, in update_flow_placement
    self.calc_path_tree_dijkstras(groupflow_trace_event)
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 128, in calc_path_tree_dijkstras
    self._calc_link_weights()
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 200, in _calc_link_weights
    sys.float_info.max / core.openflow_flow_tracker.get_num_tracked_links())
ZeroDivisionError: float division by zero
WARNING:openflow.igmp_manager:Router down: 00-00-00-00-00-06
WARNING:openflow.groupflow:Multicast topology changed, recalculating all paths.
WARNING:openflow.flow_tracker:PortStats unavailable for Switch: 00-00-00-00-00-08 Port: None
ERROR:core:Exception while handling OpenFlowNexus!ConnectionDown...
Traceback (most recent call last):
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 231, in raiseEventNoErrors
    return self.raiseEvent(event, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 278, in raiseEvent
    rv = event._invoke(handler, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 156, in _invoke
    return handler(self, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/openflow/igmp_manager.py", line 1119, in _handle_ConnectionDown
    self.raiseEvent(event)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 278, in raiseEvent
    rv = event._invoke(handler, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 156, in _invoke
    return handler(self, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 608, in _handle_MulticastTopoEvent
    self.multicast_paths[multicast_addr][source].update_flow_placement(groupflow_trace_event)
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 375, in update_flow_placement
    self.calc_path_tree_dijkstras(groupflow_trace_event)
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 128, in calc_path_tree_dijkstras
    self._calc_link_weights()
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 200, in _calc_link_weights
    sys.float_info.max / core.openflow_flow_tracker.get_num_tracked_links())
ZeroDivisionError: float division by zero
WARNING:openflow.igmp_manager:Link down: 00-00-00-00-00-08 -> 00-00-00-00-00-07
WARNING:openflow.groupflow:Multicast topology changed, recalculating all paths.
WARNING:openflow.flow_tracker:PortStats unavailable for Switch: 00-00-00-00-00-08 Port: None
ERROR:core:Exception while handling Discovery!LinkEvent...
Traceback (most recent call last):
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 231, in raiseEventNoErrors
    return self.raiseEvent(event, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 278, in raiseEvent
    rv = event._invoke(handler, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 156, in _invoke
    return handler(self, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/openflow/igmp_manager.py", line 1033, in _handle_LinkEvent
    self.raiseEvent(MulticastTopoEvent(MulticastTopoEvent.LINK_DOWN, link_changes, self.adjacency))
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 278, in raiseEvent
    rv = event._invoke(handler, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 156, in _invoke
    return handler(self, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 608, in _handle_MulticastTopoEvent
    self.multicast_paths[multicast_addr][source].update_flow_placement(groupflow_trace_event)
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 375, in update_flow_placement
    self.calc_path_tree_dijkstras(groupflow_trace_event)
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 128, in calc_path_tree_dijkstras
    self._calc_link_weights()
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 200, in _calc_link_weights
    sys.float_info.max / core.openflow_flow_tracker.get_num_tracked_links())
ZeroDivisionError: float division by zero
WARNING:openflow.igmp_manager:Router down: 00-00-00-00-00-07
WARNING:openflow.groupflow:Multicast topology changed, recalculating all paths.
WARNING:openflow.flow_tracker:PortStats unavailable for Switch: 00-00-00-00-00-08 Port: None
ERROR:core:Exception while handling OpenFlowNexus!ConnectionDown...
Traceback (most recent call last):
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 231, in raiseEventNoErrors
    return self.raiseEvent(event, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 278, in raiseEvent
    rv = event._invoke(handler, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 156, in _invoke
    return handler(self, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/openflow/igmp_manager.py", line 1119, in _handle_ConnectionDown
    self.raiseEvent(event)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 278, in raiseEvent
    rv = event._invoke(handler, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/lib/revent/revent.py", line 156, in _invoke
    return handler(self, *args, **kw)
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 608, in _handle_MulticastTopoEvent
    self.multicast_paths[multicast_addr][source].update_flow_placement(groupflow_trace_event)
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 375, in update_flow_placement
    self.calc_path_tree_dijkstras(groupflow_trace_event)
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 128, in calc_path_tree_dijkstras
    self._calc_link_weights()
  File "/usr/local/home/cse222a05/pox/pox/openflow/groupflow.py", line 200, in _calc_link_weights
    sys.float_info.max / core.openflow_flow_tracker.get_num_tracked_links())
ZeroDivisionError: float division by zero
WARNING:openflow.igmp_manager:Link down: 00-00-00-00-00-08 -> 00-00-00-00-00-0a
WARNING:openflow.groupflow:Multicast topology changed, recalculating all paths.
WARNING:openflow.groupflow:Path could not be determined for receiver 00-00-00-00-00-01 (network is not fully connected)
WARNING:openflow.groupflow:Path could not be determined for receiver 00-00-00-00-00-06 (network is not fully connected)
WARNING:openflow.groupflow:Path could not be determined for receiver 00-00-00-00-00-07 (network is not fully connected)
WARNING:openflow.groupflow:Path could not be determined for receiver 00-00-00-00-00-07 (network is not fully connected)
WARNING:openflow.groupflow:Could not get connection for router: 00-00-00-00-00-01
WARNING:openflow.groupflow:Could not get connection for router: 00-00-00-00-00-06
WARNING:openflow.groupflow:Could not get connection for router: 00-00-00-00-00-07
WARNING:openflow.igmp_manager:Router down: 00-00-00-00-00-08
WARNING:openflow.groupflow:Multicast topology changed, recalculating all paths.
WARNING:openflow.groupflow:Path could not be determined for receiver 00-00-00-00-00-01 (network is not fully connected)
WARNING:openflow.groupflow:Path could not be determined for receiver 00-00-00-00-00-06 (network is not fully connected)
WARNING:openflow.groupflow:Path could not be determined for receiver 00-00-00-00-00-07 (network is not fully connected)
WARNING:openflow.groupflow:Path could not be determined for receiver 00-00-00-00-00-07 (network is not fully connected)
WARNING:openflow.groupflow:Could not get connection for router: 00-00-00-00-00-01
WARNING:openflow.groupflow:Could not get connection for router: 00-00-00-00-00-06
WARNING:openflow.groupflow:Could not get connection for router: 00-00-00-00-00-07
WARNING:openflow.igmp_manager:Router down: 00-00-00-00-00-09
WARNING:openflow.groupflow:Multicast topology changed, recalculating all paths.
WARNING:openflow.groupflow:Path could not be determined for receiver 00-00-00-00-00-01 (network is not fully connected)
WARNING:openflow.groupflow:Path could not be determined for receiver 00-00-00-00-00-06 (network is not fully connected)
WARNING:openflow.groupflow:Path could not be determined for receiver 00-00-00-00-00-07 (network is not fully connected)
WARNING:openflow.groupflow:Path could not be determined for receiver 00-00-00-00-00-07 (network is not fully connected)
WARNING:openflow.groupflow:Could not get connection for router: 00-00-00-00-00-01
WARNING:openflow.groupflow:Could not get connection for router: 00-00-00-00-00-06
WARNING:openflow.groupflow:Could not get connection for router: 00-00-00-00-00-07
WARNING:openflow.igmp_manager:Router down: 00-00-00-00-00-0a
WARNING:openflow.groupflow:Multicast topology changed, recalculating all paths.
WARNING:openflow.groupflow:Path could not be determined for receiver 00-00-00-00-00-01 (network is not fully connected)
WARNING:openflow.groupflow:Path could not be determined for receiver 00-00-00-00-00-06 (network is not fully connected)
WARNING:openflow.groupflow:Path could not be determined for receiver 00-00-00-00-00-07 (network is not fully connected)
WARNING:openflow.groupflow:Path could not be determined for receiver 00-00-00-00-00-07 (network is not fully connected)
WARNING:openflow.groupflow:Could not get connection for router: 00-00-00-00-00-01
WARNING:openflow.groupflow:Could not get connection for router: 00-00-00-00-00-06
WARNING:openflow.groupflow:Could not get connection for router: 00-00-00-00-00-07
WARNING:openflow.igmp_manager:Router down: 00-00-00-00-00-0b
WARNING:openflow.groupflow:Multicast topology changed, recalculating all paths.
WARNING:openflow.groupflow:Path could not be determined for receiver 00-00-00-00-00-01 (network is not fully connected)
WARNING:openflow.groupflow:Path could not be determined for receiver 00-00-00-00-00-06 (network is not fully connected)
WARNING:openflow.groupflow:Path could not be determined for receiver 00-00-00-00-00-07 (network is not fully connected)
WARNING:openflow.groupflow:Path could not be determined for receiver 00-00-00-00-00-07 (network is not fully connected)
WARNING:openflow.groupflow:Could not get connection for router: 00-00-00-00-00-01
WARNING:openflow.groupflow:Could not get connection for router: 00-00-00-00-00-06
WARNING:openflow.groupflow:Could not get connection for router: 00-00-00-00-00-07
WARNING:openflow.igmp_manager:Router down: fe-e9-eb-5e-90-45
WARNING:openflow.groupflow:Multicast topology changed, recalculating all paths.
WARNING:openflow.groupflow:Path could not be determined for receiver 00-00-00-00-00-01 (network is not fully connected)
WARNING:openflow.groupflow:Path could not be determined for receiver 00-00-00-00-00-06 (network is not fully connected)
WARNING:openflow.groupflow:Path could not be determined for receiver 00-00-00-00-00-07 (network is not fully connected)
WARNING:openflow.groupflow:Path could not be determined for receiver 00-00-00-00-00-07 (network is not fully connected)
WARNING:openflow.groupflow:Could not get connection for router: 00-00-00-00-00-01
WARNING:openflow.groupflow:Could not get connection for router: 00-00-00-00-00-06
WARNING:openflow.groupflow:Could not get connection for router: 00-00-00-00-00-07
